README.md

```diff
@@ -139,10 +139,18 @@ jasentool converge --output_dir /path/to/jasen/assets/tbdb
 ### Create tbdb (ensure tb-profiler is installed)
 
 ```
+cd /path/to/jasen/assets/tbdb
```

Comment:
This is fine, but I have started to wonder lately if we should change the `/path/to/jasen` to `$JASENPATH` or similar in various places, to make the commands more copy-and-pastable?

Reply:
Good idea. Do we want it now or?

Reply:
Feel free to push changes

Reply:
Can do it later for sure. Better to change it everywhere in one Go in a separate PR I guess.

Reply:
Good idea!